modules/memegenBidAdapter.js

```diff
@@ -75,9 +76,9 @@ var MemeGenAdapter = function MemeGenAdapter() {
           secure: secureValue()
         }],
         app: {
-          id: publisherId,
+          id: toStringIfExists(utils.getBidIdParameter('appId', bidReq.params) || appBundle),
```

Comment:
@royasoolin we can add an appID to the phpData, or pass the bundle
this is supposed to be "our" app ID (it's optional anyhow)